src/backend/launcher.ts

```diff
@@ -237,7 +238,7 @@ async function prepareLaunch(
       }
     }
     // for native games lets use scout for now
-    const runtimeType = isNative ? 'scout' : nonNativeRuntime
+    const runtimeType = isNative ? 'sniper' : nonNativeRuntime
```

Comment:
We had discussion about this on Discord, switching it to sniper or soldier instead of old scout, should allow for better game compatibility (Linux native). Let's have it this way until we build our own runtime based on pressure-vessel